src/utils/roundNumber.js

```diff
@@ -0,0 +1,7 @@
+export default function toFixed(value) {
```

Comment:
This is cool, although lodash has a `round` function that does the same thing. :)